lib/cbus_elixir/app/app.ex

```diff
@@ -103,4 +103,66 @@ defmodule CbusElixir.App do
   def change_speaker(%Speaker{} = speaker) do
     Speaker.changeset(speaker, %{})
   end
+
+  alias CbusElixir.App.Attendee
```

Comment:
`alias` should always go to the top.

However, I think I understand your wanting to keep it separate. It relates to the methods you define right below... 

Ultimately I think we should move these methods out of `CbusElixir.App`. and into `CbusElixir.Query` (I think the name makes more sense). From there we can split out a submodule for `CbusElixir.Query.Attendee` and you can insert your added code there.

What do you think?

Reply:
Oh I didn't mean another app. I was just talking about another module.

I'm fine for leaving it for now. It's not too big yet